pom.xml

```diff
@@ -377,38 +377,32 @@
 
         <plugins>
             <plugin>
-                <groupId>org.kaazing</groupId>
+                <groupId>org.codehaus.mojo</groupId>
                 <artifactId>license-maven-plugin</artifactId>
-                <version>1.0.0.11</version>
+                <version>1.10</version>
+                <inherited>false</inherited>
+                <configuration>
+                    <outputDirectory>target</outputDirectory>
+                    <thirdPartyFilename>NOTICE.txt</thirdPartyFilename>
```

Comment:
Should the top level Notice.txt file be checked against this generated one to see if it is up to date?